runtimes/common/src/constants.rs

```diff
@@ -408,6 +408,18 @@ pub mod preimage {
 	}
 }
 
+pub mod tips {
+	use super::*;
+
+	parameter_types! {
+		pub const DataDepositPerByte: Balance = deposit(0, 1);
+		pub const MaximumReasonLength: u32 = 16384;
+		pub const TipCountdown: BlockNumber = DAYS;
+		pub const TipFindersFee: Percent = Percent::from_percent(20);
+		pub const TipReportDepositBase: Balance = deposit(1, 1);
```

Comment:
In Polkadot, this is set to 1 Dollar. However, I wanted to align with our other deposits.